src/utils/constants/index.js

```diff
@@ -15,4 +15,10 @@ export const CONTROL_WITH_CONFIG = {
 };
 // Constants for unknown and not-applicable values
 export const UNKNOWN = 'unknown';
-export const NA = 'NA';
\ No newline at end of file
+export const NA = 'NA';
+// Integration types
+export const GA_TO_SPLIT = 'GA_TO_SPLIT';
```

Comment:
Question: Why not a constants file in integrations folder? because it's used in the settings too?

Reply:
Yes, like storage related constants.